src/index.ts

```diff
@@ -15,26 +15,51 @@ module.exports = {
     backend: `corvid-types/${TS_CONFIG_PATHS.BACKEND}`,
     public: `corvid-types/${TS_CONFIG_PATHS.PUBLIC}`
   },
+  baseLibs: {
```

Comment:
IMO we should have an API that exposes compilerOptions by context to code-code.
Except that everything looks good

Reply:
Not sure I agree... Let's talk about it.